pkg/band/band.go

```diff
@@ -173,6 +173,9 @@ type Band struct {
 	// DefaultMaxEIRP in dBm
 	DefaultMaxEIRP float32
 
+	// LoRaCodingRate is the coding rate used for LoRa modulation.
+	LoRaCodingRate string
```

Comment:
While we're at it, we should perhaps also add the sync word

Reply:
Let's do that when we need it and there's a way to control it.

- UDP packet forwarder flavors do not support setting the sync word not a `public` flag
- Basic Station does not support setting the sync word either, but does allow for `lorawan_public`, probably changing the sync word
- The Things Gateway (Kickstarter) supports setting the sync word but 

When things converge, we'll see what we do